source/NVDAObjects/__init__.py

```diff
@@ -949,6 +949,7 @@ def event_mouseMove(self,x,y):
 

```

Comment:
Could you please add uniqueID to the base NVDAObject, along with a doc string saying what it is supposed to be? I guess for the vanilla NVDAObject, hash(self) might suffice.